templates/course_homepage/course_homepage.css

```
@@ -0,0 +1,263 @@
+body,html{
```

```suggestion
body,
html {
```
One CSS selector in each line, comma-separated, and a space before the starting curly brace.